controllers/users.js

```diff
@@ -201,6 +201,23 @@ const getUsernameAvailabilty = async (req, res) => {
   }
 };
 
+const getUsername = async (req, res) => {
```

Comment:
I'd suggest 
```suggestion
const generateUsername = async (req, res) => {
```

Reply:
Done with changes can you please check latest commit